compliance/cis_k8s/rules/cis_1_2_18/test.rego

```diff
@@ -4,7 +4,7 @@ import data.kubernetes_common.test_data
 import data.lib.test
 
 test_violation {
-	test.assert_fail(finding) with input as rule_input("--insecure-bind-address=")
+	test.assert_fail(finding) with input as rule_input("--insecure-bind-address=0.0.0.0")
```

Comment:
Maybe I missed something, but according to the official documentation, the value of the --insecure-bind-address should be a string.

Speaking about the rule - what the Rego should do if the flag exists and has no value? 
I mean if you didn't define the insecure bind address what is the violation here?

I read in the benchmark, that we need to check that the flag doesn't exist, but what does it mean to have the flag with no value - I don't know what the violation here.

WDT ?

Reply:
if the flag exists it means that it will not use the default value from my understanding, so just having the flag there should account for a violation, but I'm not entirely sure about it. 

> Maybe I missed something, but according to the official documentation, the value of the --insecure-bind-address should be a string.

it can still be an empty string.

> Speaking about the rule - what the Rego should do if the flag exists and has no value?
I mean if you didn't define the insecure bind address what is the violation here?

the violation here, per the benchmark definition, "Verify that the --insecure-bind-address argument does not exist."

> I read in the benchmark, that we need to check that the flag doesn't exist, but what does it mean to have the flag with no value - I don't know what the violation here.

This is exactly why we added this test, to follow the benchmark definition. if you want to remove it please provide something to prove that it's not relevant.